st2common/st2common/transport/announcement.py

```diff
@@ -0,0 +1,74 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from kombu import Exchange, Queue
+
+from st2common import log as logging
+from st2common.constants.trace import TRACE_CONTEXT
+from st2common.models.api.trace import TraceContext
+from st2common.transport import publishers
+from st2common.transport import utils as transport_utils
+
+LOG = logging.getLogger(__name__)
+
+# Exchange for Announcements
+ANNOUNCEMENT_XCHG = Exchange('st2.announcement', type='topic')
+
+
+class AnnouncementPublisher(object):
+    def __init__(self, urls):
+        self._publisher = publishers.PoolPublisher(urls=urls)
+
+    def publish(self, payload=None, routing_key=None):
```

Comment:
No routing key is problematic (and I'm pretty sure this is not the only place where it happens in our code since I basically copypasted it before I realized messages just doesn't hit the queue with `None` as a routing key.

Empty payload might be considered a ping, but yeah, no reason to sacrifice the clarity with this one.